### DISCUSSION TITLE
Where can I read more information about how Python code gets evaluated

### DISCUSSION THREAD

**Discussion by maxkoretskyi** [wasmerio/wasmer#4800] [General] [answered]
I'm considering wasmer as a secured runtime for user generated Python code. I've run through the basic example that uses [python/python](https://wasmer.io/python/python) package:

```
$ wasmer run python/python -- -c "print(1+1)"
2
```

this works which is great. 

I'd like to learn more about the inner workings to figure out whether it can support Python scripts that use libraries with C modules/extensions, e.g. Pandas.

How do you suggest I approach this learning path?

**Comment by syrusakbary** [comment, marked as answer]
Hi! We just compiled the normal Python into WASIX. Right now we don't support Native modules, but we are working on it.
Stay tuned!
